content/newslist.dat

```diff
@@ -1,3 +1,7 @@
+**[04/2024]** Wu-Jung and Valentina presented two talks at the [WGFAST 2024](https://www-iuem.univ-brest.fr/wgfast/?lang=en) meeting in France on pipelines for echosounder data processing (on ship & cloud).

```

Comment:
```suggestion
**[04/2024]** Wu-Jung and Valentina presented two talks at the [WGFAST 2024](https://www-iuem.univ-brest.fr/wgfast/?lang=en) meeting in France on pipelines and software tools for echosounder data processing on both ship and cloud.
```
We can probably create pages with abstract and maybe 1 fig and link from the news.

Reply:
Should we just link to the slides? Seems a lot of pages to make for each abstract? We can also link to the slides from the individual project descriptions.

Reply:
Now that I am looking through the website I realize that we had a whole talks section. Ok, I will make pages, but maybe in the future we reorganize a bit so those are linked to the projects, and people can read more on the projects.